### ISSUE TITLE
slim resnet batchnorm missing

### ISSUE THREAD

**Issue by timbmg** [tensorflow/tensorflow#23555]
I just did a MWE of resnet v1 (see code below). However, when I inspect the graph in tensorboard, I could not see the batch norm layers.
```
import tensorflow as tf
import tensorflow.contrib.slim as slim
import tensorflow.contrib.slim.nets 
import numpy as np

input = tf.placeholder(shape=[None, 224, 224, 3], dtype=tf.float32)
resnet = tf.contrib.slim.nets.resnet_v1.resnet_v1_50(input, 1000, is_training=True)
tf.summary.histogram("resnet", resnet[0])

init = tf.global_variables_initializer()
with tf.Session() as sess:
    sess.run(init)
    
    train_writer = tf.summary.FileWriter( './tflogs ', sess.graph)
    
    merge = tf.summary.merge_all()
    
    summary, out = sess.run([merge, resnet], feed_dict={input: np.random.random((2, 224, 224, 3))})
    
    train_writer.add_summary(summary)
```

**Comment by sguada** [comment]
You should use the resnet_arg_scope see
https://github.com/tensorflow/tensorflow/blob/9590c4c32dd4346ea5c35673336f5912c6072bf2/tensorflow/contrib/slim/python/slim/nets/resnet_v1.py#L43

**Comment by sguada** [comment]
```
input = tf.placeholder(shape=[None, 224, 224, 3], dtype=tf.float32)
with tf.contrib.slim.arg_scope(tf.contrib.slim.nets.resnet_v1. resnet_arg_scope()):
  resnet = tf.contrib.slim.nets.resnet_v1.resnet_v1_50(input, 1000, is_training=True)
```
